tests/kani/Intrinsics/AlignOfVal/fixme_align_of_fat_ptr.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: --unwind 3
+
+//! This test case checks the behavior of align_of_val for traits.
+#![allow(dead_code)]
+
+use std::mem::align_of_val;
+
+trait T {}
+
+struct A {
+    id: u128,
+}
+
+impl T for A {}
+
+#[cfg_attr(kani, kani::proof)]
+fn check_align_simple() {
+    let a = A { id: 0 };
+    let t: &dyn T = &a;
+    assert_eq!(align_of_val(t), 8);
```

Comment:
Does this assume a particular architecture?

Reply:
This should hold for the architecture we currently support.